src/app/modules/group/http-services/get-group-members.service.ts

```diff
@@ -21,8 +21,8 @@ export interface Member {
   id: string,
   memberSince: Date|null;
   action: 'invitation_accepted'|'join_request_accepted'|'joined_by_code'|'added_directly';
-  user: {
-    login: string|null,
+  user: null|{
```

Comment:
Wasn't this done in the previous PR?

Reply:
When I rebase the previous PR, some commits were not included :/
I did another PR with the missing commits #209